### PR TITLE
Fix: Azure silo fails to start under debugger

### LINKED ISSUE #1422: CodeGeneratorManager fails to initialize if OrleansCodeGenerator.dll is not already loaded
Discovered this wile debugging an Azure silo.

### PR DESCRIPTION
**Pull request by ca-ta**
Fixes #1422

### CODE REVIEW

**Comment by jthelin** [comment]
This is CodeGen related, so do you want to take a look @ReubenBond ?

File: src/OrleansRuntime/Silo/Silo.cs [hunk 1]
```diff
@@ -166,7 +166,6 @@ internal Silo(string name, SiloType siloType, ClusterConfiguration config, ILoca
             StatisticsCollector.Initialize(nodeConfig);
             
             SerializationManager.Initialize(globalConfig.UseStandardSerializer, globalConfig.SerializationProviders, globalConfig.UseJsonFallbackSerializer);
-            CodeGeneratorManager.GenerateAndCacheCodeForAllAssemblies();
```
Comment (sergeybykov) [inline src/OrleansRuntime/Silo/Silo.cs]: Why remove this line?

**Comment by sergeybykov** [comment]
I'm starting to get uncomfortable with automatically assigning anything codegen related to @ReubenBond. :-) And I'm guilty of that myself.

File: src/OrleansRuntime/Silo/Silo.cs (hunk 1 above)
Comment (ca-ta) [inline src/OrleansRuntime/Silo/Silo.cs]: The line is redundant (not present  in client initialization either) - the call is already made as part of the `SerializationManager` [initialization](https://github.com/ca-ta/orleans/blob/codegen_initialization/src/OrleansRuntime/Silo/Silo.cs#L168), [here](https://github.com/ca-ta/orleans/blob/codegen_initialization/src/Orleans/AssemblyLoader/AssemblyProcessor.cs#L70).

**Comment by sergeybykov** [comment]
Thank you, @ca-ta!

**Pull request merged**